scripts/imu_to_pose_node.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import rclpy
+import rclpy.logging
+from geometry_msgs.msg import PoseStamped
+from rclpy.node import Node
+from rclpy.qos import HistoryPolicy, ReliabilityPolicy, QoSProfile
+from sensor_msgs.msg import Imu
+
+
+class ImuToPoseNode(Node):
+    """
+    Subscribe to the /mavros/imu/data message, get the orientation, and republish it as a
+    PoseStamped message. This is useful for comparing the Pixhawk and x150 outputs in rviz2.
+    """
+
+    def __init__(self):
+        super().__init__('imu_to_pose')
+
+        imu_qos = QoSProfile(reliability=ReliabilityPolicy.BEST_EFFORT,
+                             history=HistoryPolicy.SYSTEM_DEFAULT)
+
+        self._imu_sub = self.create_subscription(Imu, '/mavros/imu/data',
+                                                 self.imu_callback, imu_qos)
+
+        self._pose_pub = self.create_publisher(PoseStamped, 'imu_pose', 10)
+
+    def imu_callback(self, imu_msg: Imu):
```

Comment:
Should add a comment that the position attributes of the message are intentionally being left as zero, because we don't have any valid position information.